data/json/regional_map_settings.json

```diff
@@ -108,6 +108,7 @@
           "houses": {
               "house_two_story_basement": 1,
               "house": 1000,
+              "house_prepper": 30,
```

Comment:
Are weights consistent?

Reply:
Most probably, there are 35 houses in `\data\json\mapgen\house` folder with total weight of 1000.